app/Judite/Models/Solver.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Judite\Models;
+
+
+use Illuminate\Support\Collection;
+use GuzzleHttp\Client;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+
+class solver extends Model
+{
+	/**
+	 * changes the exchange format to the one used by the solver
+	 *
+	 * @param Course								$course
+	 */
+	public static function SolveAutomicExchangesOfCourse(Course $course)
+	{
+        $client = new Client();
+		$url = 'http://10.0.2.2:4567/';
```

Comment:
```suggestion
		$url = 'http://10.0.2.2:4567/';
```

This url should be moved to the .env file